src/modules/add-task.js

```diff
@@ -0,0 +1,17 @@
+import { tasks } from './task'
+import { DisplayList } from '../index';
+import { Task } from './task'
+
+const addTask = document.getElementById('add-task');
+const submitBtn = document.querySelector('.submit-button');
+
+submitBtn.addEventListener('click', function() {
```

Comment:
kindly. In this line, I recommend using the function Arrow. This helps you to have your code more tidy and short.